pennylane/devices/default_qubit.py

```diff
@@ -503,8 +503,8 @@ def expval(self, observable, shot_range=None, bin_size=None):
                 # whereas the .coeff attribute will always be the same input dtype that the user provided.
                 for op, coeff in zip(observable.ops, observable.data):
 
-                    # extract a scipy.sparse.coo_matrix representation of this Pauli word
-                    coo = qml.operation.Tensor(op).sparse_matrix(wires=self.wires)
+                    # extract a scipy.sparse.csr_matrix representation of this Pauli word
+                    coo = qml.operation.Tensor(op).sparse_matrix(wires=self.wires, format="coo")
```

Comment:
Why the `coo` format is still used here? Is it because `csr` is used only for the expval computation of the full sparse Hamiltonian matrix?